### ISSUE TITLE
suggested test case

### ISSUE THREAD

**Issue by jedwards4b** [NCAR/iCESM1.3_iHESP_hires#16]
@nusbaume Can you suggest a (small) test case to try on gust?

**Comment by nusbaume** [comment]
@jedwards4b  Sure, but if you give me a couple days I can see if I can turn it into a cheaper, "official" test using CIME's testing framework.  I'll let you know when I have the (isotope-enabled) test up and working.

**Comment by jedwards4b** [comment]
Please make sure that the test case is relevant for the ASD project.   Are you aware of anyone else providing software support for this ASD project?   Does Jiang Zhu have a github account?

**Comment by nusbaume** [comment]
Yes, Jiang is on Github, his username is @jiang-zhu.  I also remember being on an email back in April stating that Siddhartha Ghosh (@siddsghosh) from CISL would also be available to help, although I am not sure if that was specific for this project or ASD in general.

Also, that's a good point about having it be ASD-specific.  @jiang-zhu, do you have a specific (low resolution) simulation that you have been using to test the model?  If so, can you either provide the path to a case here, or provide the steps you do to set it up?  Thanks!

**Comment by jiang-zhu** [comment]
@jedwards4b @nusbaume Thank you for your help!

Here is a document on Cheyenne with my steps to create a testing case in ne30_g16. 
/glade/u/home/jiangzhu/cesm/cases/test.csh

If it is needed for testing purposes, I can try to create a lower-resolution case.

Our ASD project will use the resolution of ne120_t12.

**Comment by jedwards4b** [comment]
@jiang-zhu Attempting to port to the gust system, I am running into a runtime error:
```
158  gu000gu0001.hsn.gu.hpc.ucar.edu 0: MCT::m_AttrVect::indexRA_:: FATAL--attribute not found: "Sl_qref_16O" Traceback:  
|X|MCT::m_AttrVect::indexRA_
000.MCT(MPEU)::die.: from MCT::m_AttrVect::indexRA_()
MPICH ERROR [Rank 0] [job id 186cc95f-24d8-417b-b937-dbef1edf2ad7] [Wed Aug 31 12:29:23 2022] [gu0001] - Abort(2) (rank 0 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 2) - process 0

```
any ideas?

**Comment by jiang-zhu** [comment]
Perhaps, try
./xmlchange FLDS_WISO=TRUE
before case.setup?

**Comment by jedwards4b** [comment]
Yes sorry, I thought I had done that but missed it.

**Comment by nusbaume** [comment]
FYI I just pushed some changes requested by Jiang, but they were all just making some of the isotope namelist settings true by default (so they hopefully won't impact the gust work at all).

**Comment by jedwards4b** [comment]
Now getting an error:

**Comment by jedwards4b** [comment]
Jesse - can you give me push permissions to each of these repositories?  I would like to open PRs for the gust port and don't really want to deal with forking.

**Comment by nusbaume** [comment]
Sure, you should have write access to all of the relevant repos now.  Of course if I missed one please let me know.
